### DISCUSSION TITLE
Error: the function of ecg_intervalrelated()

### DISCUSSION THREAD

**Discussion by RyotaTKN** [neuropsychology/NeuroKit#656] [Q&A] [answered]
Hello, 

How can I deal with the error after running following code?

```
analyze_epochs_inter = nk.ecg_intervalrelated(epochs)
```

Thank you in advance!

![image](https://user-images.githubusercontent.com/58549690/173239307-682c68ce-40bc-4c3e-a688-c962508a5086.png)

**Comment by DominiqueMakowski** [comment, marked as answer]
Transferred to #657
